go.mod

```diff
@@ -4,6 +4,8 @@ go 1.23.8
 
 toolchain go1.23.10
 
+replace github.com/crossplane/crossplane-runtime => github.com/negz/crossplane-runtime v0.0.0-20250618002755-794eae126b22
```

Comment:
This is https://github.com/crossplane/crossplane-runtime/pull/849. Can't merge until that's merged.

Reply:
Updated.